### PR TITLE
Add reasons for not sprayed structures to map indicators list

### LINKED ISSUE #1069: RVL-1027 - Add Indicator with Reasons for not-sprayed structures
1. Go to the Zambia Reveal WebUI > Select Monitor > select IRS Reporting > Select a completed IRS Plan. On the dashboard table that comes up, drill down to the lowest level:
- On the indicators list to the right hand side of the map, add an indicator named `Reasons for not-sprayed structures` . This would read e.g. 7 out of 100 structures not sprayed. The different reasons given for not sprayed will then be given as aggregates e.g.   Sick - 1; Locked - 0,  Funeral - 0; Refused - 2; Adult not present - 4; Other - 0

### PR DESCRIPTION
**Pull request by ciremusyoka**
Closes #1069 
Add a list of reason why some structures were not sprayed and an over view of the number of structured not sprayed.

### CODE REVIEW

**Comment by ciremusyoka** [comment]
ScreenShot
Screenshot from 2020-07-29 16-56-47

File: src/store/ducks/generic/jurisdictions.ts [hunk 1]
```diff
@@ -58,6 +59,8 @@ export interface ZambiaIRSFocusArea extends GenericJurisdiction {
   spraytarg: number;
   spraysuccess: number;
   roomcov: number;
+  notsprayed_reasons: string[];
+  notsprayed_reasons_counts: Dictionary;
```
Comment (moshthepitt) [inline src/store/ducks/generic/jurisdictions.ts:63]: I think that we can make this type more accurate.  Seems like it is Dictionary<number>

**Comment by ciremusyoka** [comment]
> based on the screenshot, the number of not-sprayed structures should be `16 of 72`. You can arrive at 16 in one of two ways:

Have updated this calculation.

File: src/containers/pages/IRS/Map/index.tsx [hunk 2]
```diff
@@ -255,6 +256,20 @@ const IRSReportingMap = (props: IRSReportingMapProps & RouteComponentProps<Route
 
   const gisidaWrapperProps = getGisidaWrapperProps(jurisdiction, structures, indicatorStops);
 
+  /**
+   * Create list elements from dictionary
+   * @param {Dictionary} dict
+   */
+  const processListDisplay = (dict: Dictionary | string) => {
+    const createList = (obj: Dictionary) =>
+      Object.keys(obj).map(key => (
```
Comment (moshthepitt) [inline src/containers/pages/IRS/Map/index.tsx:265]: Can we use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)?  Is it better?

Comment (ciremusyoka) [reply src/containers/pages/IRS/Map/index.tsx]: Yah! it is better.

File: src/containers/pages/IRS/Map/index.tsx [hunk 3]
```diff
@@ -255,6 +256,20 @@ const IRSReportingMap = (props: IRSReportingMapProps & RouteComponentProps<Route
 
   const gisidaWrapperProps = getGisidaWrapperProps(jurisdiction, structures, indicatorStops);
 
+  /**
+   * Create list elements from dictionary
+   * @param {Dictionary} dict
+   */
+  const processListDisplay = (dict: Dictionary | string) => {
+    const createList = (obj: Dictionary) =>
+      Object.entries(obj).map(keyVal => (
```
Comment (p-netm) [inline src/containers/pages/IRS/Map/index.tsx:265]: nitpick: `Object.entries().map(([key, value]) => {})`

**Pull request merged**